packages/@aws-cdk/aws-glue/README.md

```diff
@@ -23,6 +23,22 @@
 
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+## Connection
+
+A `Connection` allows Glue jobs, crawlers and development endpoints to access certain types of data stores.
+
+```ts
+new glue.Connection(stack, 'MyConnection', {
+  connectionType: glue.ConnectionTypes.NETWORK,
+  securityGroups: [securityGroup],
+  subnet,
```

Comment:
If we have in the example, lets just add a sentence or two into what these are - and why they are needed.

Reply:
Basing my comments on Subnet/Security groups section in https://docs.amazonaws.cn/en_us/glue/latest/dg/connection-defining.html